src/blockchain/log-processors/order-created.ts

```diff
@@ -53,17 +51,17 @@ export function processOrderCreatedLog(db: Knex, augur: Augur, log: FormattedEve
         sharesEscrowed: augur.utils.convertOnChainAmountToDisplayAmount(sharesEscrowed, tickSize).toFixed(),
       };
       const orderId = { orderId: log.orderId };
-      checkForOrphanedOrders(db, orderData, (err) => {
+      db.select("marketId").from("orders").where(orderId).asCallback((err: Error|null, ordersRows?: Array<Partial<OrdersRow<BigNumber>>>): void => {
         if (err) return callback(err);
-        db.select("marketId").from("orders").where(orderId).asCallback((err: Error|null, ordersRows?: Array<Partial<OrdersRow<BigNumber>>>): void => {
+        let upsertOrder: QueryBuilder;
+        if (!ordersRows || !ordersRows.length) {
+          upsertOrder = db.insert(Object.assign(orderData, orderId)).into("orders");
+        } else {
+          upsertOrder = db.from("orders").where(orderId).update(orderData);
```

Comment:
I know you didn't change this line, but how would we update on order created?

Reply:
I assumed there was some reason for this beyond my understanding actually...